Makefile.inc

```diff
@@ -0,0 +1,3 @@
+ifndef BUILD_COMMAND
```

Comment:
Minor nit: The `ifndef` is redundant as make already supports conditionally setting variables with the `?=` operator.
See https://www.gnu.org/software/make/manual/make.html#Setting